pages/home.jsx

```diff
@@ -60,6 +60,22 @@ class Home extends React.Component {
               <img src="/assets/images/mozilla-festival_website-theme-update.svg" width="250" className="my-4" />
               <h1>Your Data and You</h1>
               <p>This year at Mozilla Festival, dive into our theme “Your Data and You,” through interactive sessions, art, games, dialogues, debates, and the latest tech. We’ll explore the risks and benefits, the implications for the health of the Internet, and how we can take control of our data, our online lives, and our collective future.</p>
+              <p className="mt-4">Speaking of your data, we're here to help you shop safe for connected gifts this holiday season. Check out our *Privacy Not Included buyer's guide.</p>
```

Comment:
@mmmavis can we change the copy to be this instead?

We've even created a guide to help you take control of your data and buy safe, secure gifts this holiday season. Check out the *Privacy Not Included Buyer's Guide [online](link to buyers guide), or come visit us during the festival weekend at Ravensbourne University on Level 0 to learn more.